examples/launch_height_baselines.py

```diff
@@ -93,9 +76,39 @@
         #     metric=metric
         # ),
     ]
+    try:
+        from syne_tune.optimizer.baselines import BayesianOptimization
+        # example of setting additional kwargs arguments
+        schedulers.append(
+            BayesianOptimization(
+                config_space,
+                metric=metric,
+                mode=mode,
+                search_options={"num_init_random": n_workers + 2},
+            )
+        )
+        from syne_tune.optimizer.baselines import MOBSTER
+        schedulers.append(
+            MOBSTER(
+                config_space,
+                metric=metric,
+                resource_attr="epoch",
+                max_t=max_steps,
+                mode=mode,
+            )
+        )
+    except NameError:
+        print(
+            "Gaussian process based searchers are not "
+            "imported since dependencies are missing. You can install them with\n"
+            "   pip install 'syne-tune[gpsearchers]'\n"
+            "or (for everything)\n"
+            "   pip install 'syne-tune[extra]'"
+        )
+        pass
```

Comment:
nit: You do not need `pass` here as you already have a log statement in the except clause. Same applies to other `pass`es.